### ISSUE TITLE
YouTube 11.32.53 Swipe Issues

### ISSUE THREAD

**Issue by shamsul007** [blueberry6401/xposed_youtube-swipe-to-seek#13]
Latest version of YouTube, Swipe to seek feature does not function in both small view and full screen view. However volume slider does function.

Love this module! Thank you for your hard work

**Comment by blueberry6401** [comment]
Would you please check the option "Swipe to seek" is turned on or not in app's options, or turn it off and on again and check again.

**Comment by shamsul007** [comment]
The Swipe to seek is turned on. Tried turning it off and on, restarted app (module and YouTube app) and also reinstalled module. But still not working.

**Comment by BibleBeliever** [comment]
Same problem here. (MM)

**Comment by forumber** [comment]
Same here too

**Comment by shamsul007** [comment]
Are you all running the latest version of YouTube that I am? (11.32.53) and have you all tried what the developer suggested?

blueberry6401
Would you please check the option "Swipe to seek" is turned on or not in app's options, or turn it off and on again and check again.

**Comment by BibleBeliever** [comment]
Yes and yes. The new YouTube apps seem to break Swipe to Seek and YouTube Background Playback modules. Very frustrating

**Comment by shamsul007** [comment]
Latest version of YouTube background play (1.8.31) is working perfectly for me. 

It seems to me that the YouTube app goes out of its way to suck more and more.

Thank god for these devs who work hard to make the app functional.

**Comment by BibleBeliever** [comment]
I just found out why YouTube Background Playback "wasn't working": I had YouTube notifications blocked. 

Anyway, it's God*, upper case G.

**Comment by blueberry6401** [comment]
My youtube still stucks at version 11.29.55 and I get no update available.
Also I downloaded version 11.32.53 from apk-dl but I cannot install it.
I'm working to find this out.

**Comment by ggkameleon** [comment]
Yes I confirm the latest youtube block swipe I'm sad 😑 do you realese an update ? 
The only solution are to turn off/on for each swipe.

**Comment by brbsix** [comment]
@blueberry6401 Hopefully I'm not stating the obvious, but maybe the APK you downloaded was for a different architecture? You can see all the variants for the latest Youtube release [here](https://www.apkmirror.com/apk/google-inc/youtube/youtube-11-32-53-release/). I think apkmirror is better because they verify app signatures and also include a chart of all the variants.

**Comment by ggkameleon** [comment]
Hi ! Just for information please. 
You works for an update for the latest youtube ? 
It's not easy to install the before version because in my phone it's a system app & I do to exit it to the market. 
Thank you a lot 😉

**Comment by brbsix** [comment]
@ggkameleon You can disable the YouTube app to uninstall all updates, re-enable it, then sideload the update to [the last working version](https://www.apkmirror.com/apk/google-inc/youtube/youtube-11-29-55-release/). That's what I did anyways. Also remember to disable auto-update in the Play Store if you have it enabled.

Or if you have root access and want to leave auto-update enabled you can uninstall the system app (manually or with an app like Titanium Backup) and then install a resigned version of v11.29.55 (manually with jarsigner or with an app like ZipSigner or MiXplorer). That will prevent the app from ever being updated in the Play Store. Anyways, hopefully this gets fixed soon and we don't have to worry about that.

**Comment by ggkameleon** [comment]
Thank's for your support. 
Yes I'am root and I do this 🙂
I stay in this version at this time and stay an update of swipe for youtube 😎

**Comment by brbsix** [comment]
Just a FYI there was another update to the YouTube app on September 2nd, 2016. The current release is now [YouTube v11.33.58](https://www.apkmirror.com/apk/google-inc/youtube/youtube-11-33-58-release/).

**Comment by ggkameleon** [comment]
Ok thank ! Swipe work in this version v.11.33.58.?

**Comment by Celoxocis** [comment]
No it doesn't I just found out. I have that version and love swipe to seek! It's by far the best YouTube Closed Module!

**Comment by brbsix** [comment]
@ggkameleon No I just posted that for @blueberry6401 so that he can develop against the latest YouTube APK.

**Comment by blueberry6401** [comment]
Yep finally my phone got the latest version 11.33.58. I will fix this issue as fast as I can, in several days, because now I have some businesses to do first (about my job).
Thank you all.

**Comment by shamsul007** [comment]
I've downgraded to a version of YouTube that works with this module for the time being. Ridiculous how such a small module can become irreplaceable!

Take your time blueberry6401!

**Comment by ggkameleon** [comment]
@bbrix ok 
It's not cool I like more this module 😐 in ios they are a lot of tweak for jailbreak user to expend youtube ex : cercube you can swipe & back ground. 
I don't understand why in android they are not a lot of modules !? They are many user of this os and not many modules. Sniff

**Comment by ggkameleon** [comment]
Blueberry640 thank's when you can update it and notify us. 🙂

**Comment by brbsix** [comment]
I made some progress on this. It looks like the new YouTube app was intercepting and canceling horizontal swipes for some reason. I was able to prevent this but now you're unable to swipe to minimize/maximize/close the player window and instead have to rely on other buttons/gestures. But at least swipe to seek and swipe to change volume now work.

APK is available here: https://github.com/brbsix/xposed_youtube-swipe-to-seek/releases/tag/quickfix
